packages/components/src/components/assay/AssayImportPanels.tsx

```diff
@@ -259,10 +262,11 @@ class AssayImportPanelsImpl extends React.Component<Props, State> {
         const sampleColumnData = assayDefinition.getSampleColumn();
 
         if (sampleColumnData && location) {
+            const selectedSamplesFunction = this.props.loadSelections ? this.props.loadSelections : loadSelectedSamples;
```

Comment:
I'd recommend making `loadSelectedSamples` the default prop value for `loadSelections`.

```ts
static defaultProps = {
    loadSelections: loadSelectedSamples,
};
```